src/Game.js

```diff
@@ -0,0 +1,213 @@
+import React, { Component } from "react";
+import Canvas from "./Canvas";
+
+class Game extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      player1: { name: null, id: null, paddle: [235, 480] },
+      player2: { name: null, id: null, paddle: [235, 10] },
+      pongBall: [250, 250],
+      ballDirection: [-1, -1],
+      round: 0,
+      gameCode: null,
+    };
+  }
+
+  updateAnimationState = () => {
+    this.animateScene();
+    requestAnimationFrame(this.updateAnimationState);
+  };
+
+  createPlayer = (username, id, x, y) => {
+    const player = {
+      username: username,
+      id: id,
+      score: 0,
+      paddle: [x, y],
+    };
+    if (this.state.player1) {
+      this.setState({
+        player2: player,
+      });
+    } else {
+      this.setState({
+        player1: player,
+      });
+    }
+  };
+
+  createGameSettings = (x, y) => {
+    const pongBall = [x, y];
+    this.setState({ pongBall: pongBall });
+    // setGameCode(gameCode);
+  };
+
+  createGame = () => {
+    this.createGameSettings(250, 100);
+    this.createPlayer("Jon", 1, 235, 480);
+    this.createPlayer("Joe", 2, 235, 10);
+  };
+
+  startGame = () => {
+    this.createGame();
+    requestAnimationFrame(this.updateAnimationState);
```

Comment:
I can't recall whether this call to `requestAnimationFrame` is necessary since the call is made recursively in the `updateAnimationState` function. If not needed, remove to avoid confusion down the line